src/ActionModal/components/DelegationModal.vue

```diff
@@ -91,8 +76,7 @@
         />
       </TmFieldGroup>
       <span class="form-message">
-        {{ isRedelegation ? "Available to redelegate" : "Available to stake" }}
-        :
+        Available to stake:
         {{ maxAmount }}
```

Comment:
will `maxAmount` change to reflect your current delegation if you're doing a redelegation? or always shows available balance?